apps/website/src/components/ArticleCard.tsx

```diff
@@ -0,0 +1,31 @@
+import { Heading, Text, Card, CardBody, CardFooter } from "@chakra-ui/react"
+
+export type ArticleCardProps = {
+    title: string
+    minRead: number
+}
+
+export default function ArticleCard({ title, minRead }: ArticleCardProps) {
+    return (
+        <Card
+            bg={"transparent"}
```

Comment:
It looks like the background color could change. It should be a component property.